spec/schema.json

```diff
@@ -320,6 +448,70 @@
         }
       }
     },
+    "GeneralPartnerData": {
+       "type": "object",
+       "properties": {
+        "country": {
+          "type": "string"
+        },
+        "date_effective_from": {
+          "type": "string",
+          "format": "date-time"
```

Comment:
Should this be captured as day, month, year like for 'date of birth'? Can't be sure looking at registration screen, as this field doesn't appear there.

Reply:
Ok, but not sure about the format here. As we saw, maybe ISO dates will be going back and fore to the API as strings. It certainly doesn't sound right that this format includes a time portion.

Reply:
Agreed. Removed time portion